### PR TITLE
Assorted bag of fixes

### PR DESCRIPTION
**Pull request by NiLuJe**
Fairly minor stuff.

Fun fact: I get working WiFi immediately after restarting to Nickel now. Might be FW 4.8.whatever, though. Or might be gremlins, who knows.

### CODE REVIEW

**Comment by poire-z** [comment]
Just a quick note/wish after reading ```[...] the other end of this hasn't made it into
current KSM build yet.```: please try keeping koreader compatible with older KSMs (and kobo FWs)
(I still use KSM 07, and I want to keep that version as it just works, and it's the one that still has everything in /mnt/onboard/.adds/ (easier for looking/playing) and has the minimal stuff put in the root partition)

**Comment by NiLuJe** [comment]
Yep, not an issue, we have a fallback in place for anything that might be
launcher specific.

File: platform/kobo/fmon/README.txt [hunk 1]
```diff
@@ -20,4 +20,4 @@ When you update KOReader manually, it should be sufficient to simply extract the
 
 NOTE: Using the legacy version of fmon (http://www.mobileread.com/forums/showthread.php?t=218283) is *STRONGLY* discouraged.
 NOTE²: If you like the concept, and want to check out something a bit more flexible/complex, check out KFMon (https://github.com/NiLuJe/kfmon).
-
+       In which case, look at the relevant WiKi page (https://github.com/koreader/koreader/wiki/Installation-on-Kobo-devices) for more details ;).
```
Comment (Frenzie) [inline platform/kobo/fmon/README.txt]: Indentation seems a bit random?

Comment (NiLuJe) [reply platform/kobo/fmon/README.txt]: Aligned with the beginning of the previous sentence to make it clear it's the same NOTE and not an unrelated, new sentence ;).

**Comment by Frenzie** [comment]
Do you want to do some rebasing for clarity or lazy squash?

**Comment by poire-z** [comment]
Just a quick note regarding this change:
```
-[ "${WIFI_MODULE}" != "8189fs" ] && wlarm_le -i "${INTERFACE}" down
+[ "${WIFI_MODULE}" != "8189fs" ] && [ "${WIFI_MODULE}" != "8189es" ] && wlarm_le -i "${INTERFACE}" down
```

In the mark6 4.8.11073/etc/init.d/rcS, we have 81**92**es instead of 8189es:
```sh
if [ $PLATFORM == freescale ]; then
        INTERFACE=wlan0
        WIFI_MODULE=ar6000
else
        INTERFACE=eth0
        WIFI_MODULE=dhd
        if [ x$WIFI == "xRTL8189" ]; then
                WIFI_MODULE=8189fs
        elif [ x$WIFI == "xRTL8192" ]; then
                WIFI_MODULE=8192es
        fi
fi
```
although there is no such module in 4.8.11073/drivers/mx6sl-ntx/wifi/, only:
8189fs.ko  dhd.ko  sdio_wifi_pwr.ko
but may be it's 8192es too in mark7? (Haven't checked).

**Comment by NiLuJe** [comment]
That does smell like another stupid c/p mistake on my part, good catch ;)

**Pull request merged**